Refresh.GameServer/Database/GameDatabaseContext.Users.cs

```diff
@@ -380,4 +386,20 @@ public void IncrementUserFilesizeQuota(GameUser user, int amount)
             user.FilesizeQuotaUsage += amount;
         });
     }
+    
+    public void SetProfileVisibility(GameUser user, Visibility visibility)
+    {
+        this._realm.Write(() =>
+        {
+            user.ProfileVisibility = visibility;
+        });
+    }
+    
+    public void SetLevelVisibility(GameUser user, Visibility visibility)
+    {
+        this._realm.Write(() =>
+        {
+            user.LevelVisibility = visibility;
+        });
+    }
```

Comment:
These parameters technically aren't bundled in one request from the game, but to simplify the endpoint and to keep this class shorter I think it should be one function called `SetUserPrivacySettings`.

Reply:
so basically, in the implementation of the endpoint do `get current settings -> update changed fields -> update whole settings`?

Reply:
Just move the logic from the endpoint to a function in the database that applies based off of the privacy settings object. The implementation itself doesn't need to change, apart from putting everything in a write block I guess.